src/js/Framework/ToTop.js

```diff
@@ -0,0 +1,10 @@
+export class ToTop {
+
+  scrollToTop () {
```

Comment:
Gingen we dat niet verplaatsen naar https://github.com/sumocoders/FrameworkStylePackage/blob/master/src/js/Framework/Scrolling.js ?